File: Narcissistic number/Narcissistic number/Five-point-star_number.cpp
#include "Narcissistic.h"
#include "global.h"

void StarNumber() {
	for (num = 10000; num <= 100000; num++) {
		a = num / 10000;
		b = (num / 1000) % 10;
		c = (num / 100) % 10;
		d = (num / 10) % 10;
		e = num % 10;
		if (a*a*a*a*a+b*b*b*b*b+c*c*c*c*c+d*d*d*d*d+e*e*e*e*e== num)
			printf("%d\n", num);
	}
}

File: resoures/Five-point-star number.py
for num in range(10000,100000):
    a = num // 10000
    b = (num // 1000) % 10
    c = (num // 100) % 10
    d = (num // 10) % 10
    e = num % 10
    if a**5+b**5+c**5+d**5+e**5==num:
        print (num)
